src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/caseworker/event/page/SendOrReply.java

```diff
@@ -89,6 +91,15 @@ private AboutToStartOrSubmitResponse<CaseData, State> midEvent(CaseDetails<CaseD
             .build();
     }
 
+    public String reasonMessage(MessageSendDetails item) {
+        if (item.getMessageReasonList() != null && item.getMessageReasonList().getLabel() != null) {
+            return item.getMessageReasonList().getLabel();
+        } else if (item.getMessageReasonJudge() != null && item.getMessageReasonJudge().getLabel() != null) {
+            return item.getMessageReasonJudge().getLabel();
+        }
+        return null;
+    }
+
```

Comment:
Duplicate method logic already present in 'CaseworkerSendOrReply'. can we create a common method to reuse at both places?